geometry_msgs/Pose sh_pose
---
bool collision
